lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/EvaluatorRuntime.cs

```diff
@@ -95,7 +95,11 @@ public void Handle(EvaluatorControlProto message)
         {
             lock (_heartBeatManager)
             {
-                Logger.Log(Level.Info, "Handle Evaluator control message");
+                var msg = " done_evaluator = " + (message.done_evaluator == null ? "null" : "not null")
+                          + " kill_evaluator = " + (message.kill_evaluator == null ? "null" : "not null")
+                          + " stop_evaluator = " + (message.stop_evaluator == null ? "null" : "not null")
+                          + " context_control = " + (message.context_control == null ? "null" : "not null");
```

Comment:
Is it reasonable to convert these null checks into a utility method to avoid duplicating the same logic four times ?